src/GrumPHP/Task/NpmScript.php

```diff
@@ -59,7 +61,12 @@ public function run(ContextInterface $context)
             return TaskResult::createSkipped($this, $context);
         }
 
-        $arguments = $this->processBuilder->createArgumentsForCommand('npm');
+        $command = 'npm';
+        if (true === $config['is_run_task']) {
```

Comment:
This will result in `'npm run'` on the CLI instead of `'npm' 'run'`. Can you use `$arguments->addOptionalArgument()` instead?